builder/Makefile.release

```diff
@@ -0,0 +1,109 @@
+OUTDIR ?= dist
+OUT_ARCHIVE := $(OUT_DIR)/tracee.tar.gz
+OUT_CHECKSUMS := $(OUT_DIR)/checksums.txt
+RELEASE_FILES := LICENSE $(OUT_DIR)/tracee-ebpf $(OUT_DIR)/tracee-rules $(OUT_DIR)/rules $(OUT_DIR)/tracee.bpf.core.o
+PUSH_DOCKER_REPO ?= aquasec/tracee
+PUSH_DOCKER_TAG ?= $(RELEASE_TAG:v%=%)
+
+CMD_DOCKER ?= docker
+CMD_CHECKSUM ?= sha256sum
+CMD_GITHUB ?= gh
+CMD_TAR ?= tar
+CMD_CP ?= cp
+CMD_GIT ?= git
+CMD_RM ?= rm
+CMD_MAKE ?= make
+
+TARGET_CORE_IMAGE_NAME = build-alpine-tracee-core
+TARGET_BTFHUB_IMAGE_NAME = build-alpine-tracee-core-btfhub
+TARGET_NOCORE_IMAGE_NAME = build-alpine-tracee-nocore
+
+MAKE_CONTAINER = $(CMD_MAKE) -f builder/Makefile.tracee-container
```

Comment:
would you mind declaring the variables in the first (or the one that makes most sense) target that uses it ? I followed this convention so we didn't have a bunch of variables in the beginning of the Makefiles.

Reply:
The CMD_xxxx variables should be above .check_% target (without variables in between them), for uniformity.